tests/playwright/sanity/basic-gallery.test.ts

```diff
@@ -60,6 +62,34 @@ test( 'Basic Gallery Lightbox test with older Swiper', async ( { page, apiReques
 	await testBasicSwiperGallery( editor );
 } );
 
+test( 'Basic Gallery Lightbox test with breakpoints', async ( { page, apiRequests }, testInfo ) => {
+	// Arrange.
+	const wpAdmin = new WpAdminPage( page, testInfo, apiRequests );
+	const editor = await wpAdmin.openNewPage();
+	const breakpoints = new Breakpoints( page );
+
+	await editor.closeNavigatorIfOpen();
+
+	await editor.addWidget( 'image-gallery' );
+	await editor.openPanelTab( 'content' );
+	await editor.addImagesToGalleryControl();
+	await editor.setSelectControlValue( 'open_lightbox', 'yes' );
+
+	await editor.publishPage();
+
+	await breakpoints.setBreakpoint( editor, 'mobile', viewportSize.mobile.width );
+
+	// Act.
+	const pageId = await editor.getPageId();
+	await page.goto( `/?p=${ pageId }` );
+	await page.setViewportSize( viewportSize.mobile );
```

Comment:
I am thinking the following: 

1. If the mobile breakpoint is 400px, then we should test a screensize in between 400px and 767px, to ensure that the `padding: 70px` is applied. We could test a breakpoint of 500px.
2. Additionally we could test the breakpoint of 350px, to verify that the `padding: 70px 0` is applied.

Screenshot testing is sufficient, but we need to use a correct breakpoint.